src/pudl/package_data/meta/xlsx_maps/eia861/column_maps/distributed_generation_eia861.csv

```diff
@@ -2,44 +2,37 @@
-backup_capacity,-1,-1,-1,-1,-1,-1,-1,-1,6,6,6,6
-distributed_generation_capacity_owned,-1,-1,-1,-1,-1,-1,-1,-1,7,7,7,7
-internal_combustion_capacity,-1,-1,-1,-1,-1,-1,-1,-1,9,9,9,9
-combustion_turbine_capacity,-1,-1,-1,-1,-1,-1,-1,-1,10,10,10,10
-steam_turbine_capacity,-1,-1,-1,-1,-1,-1,-1,-1,11,11,11,11
-hydroelectric_capacity,-1,-1,-1,-1,-1,-1,-1,-1,12,12,12,12
-wind_capacity,-1,-1,-1,-1,-1,-1,-1,-1,13,13,13,13
-photovoltaic_capacity,-1,-1,-1,-1,-1,-1,-1,-1,14,14,14,14
-storage_capacity,-1,-1,-1,-1,-1,-1,-1,-1,15,15,15,15
-other_capacity,-1,-1,-1,-1,-1,-1,-1,-1,16,16,16,16
-total_capacity,-1,-1,-1,-1,-1,-1,-1,-1,17,17,17,17
-distillate_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-natural_gas_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-wood_capacity_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-water_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-wind_fuel_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-other_renewable_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-other_fuel_capacity,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
+generators_number,3,3,3,3,3,3,-1,-1,4,4,4,4
+generators_num_less_1_mw,4,4,4,4,4,4,3,3,-1,-1,-1,-1
+total_capacity_mw,5,5,5,5,5,5,-1,-1,5,5,5,5
+total_capacity_less_1_mw,6,6,6,6,6,6,4,4,-1,-1,-1,-1
+backup_capacity_pct,7,7,7,7,7,7,-1,-1,-1,-1,-1,-1
+estimated_or_actual_capacity_data,8,8,8,8,8,8,7,7,8,8,8,8
+internal_combustion_capacity_pct,9,9,9,9,9,9,-1,-1,-1,-1,-1,-1
+combustion_turbine_capacity_pct,10,10,10,10,10,10,-1,-1,-1,-1,-1,-1
+steam_capacity_pct,11,11,11,11,11,11,-1,-1,-1,-1,-1,-1
+hydro_capacity_pct,12,12,12,12,12,12,-1,-1,-1,-1,-1,-1
+wind_capacity_pct,13,13,13,13,13,13,-1,-1,-1,-1,-1,-1
+other_capacity_pct,14,14,14,14,14,14,-1,-1,-1,-1,-1,-1
+estimated_or_actual_tech_data,15,15,15,15,15,15,17,17,18,18,18,18
+oil_fuel_pct,16,16,16,-1,-1,-1,-1,-1,-1,-1,-1,-1
+gas_fuel_pct,17,17,17,-1,-1,-1,-1,-1,-1,-1,-1,-1
+wood_fuel_pct,18,18,18,-1,-1,-1,-1,-1,-1,-1,-1,-1
+water_fuel_pct,19,19,19,-1,-1,-1,-1,-1,-1,-1,-1,-1
+wind_fuel_pct,20,20,20,-1,-1,-1,-1,-1,-1,-1,-1,-1
+renewable_fuel_pct,21,21,21,-1,-1,-1,-1,-1,-1,-1,-1,-1
+other_fuel_pct,22,22,22,-1,-1,-1,-1,-1,-1,-1,-1,-1
+estimated_or_actual_fuel_data,23,23,23,-1,-1,-1,-1,-1,-1,-1,-1,-1
+state,-1,-1,-1,-1,16,16,18,18,3,3,3,3
+distributed_generation_owned_capacity_pct,-1,-1,-1,-1,17,17,-1,-1,-1,-1,-1,-1
+backup_capacity_mw,-1,-1,-1,-1,-1,-1,5,5,6,6,6,6
+distributed_generation_owned_capacity_mw,-1,-1,-1,-1,-1,-1,6,6,7,7,7,7
+internal_combustion_capacity_mw,-1,-1,-1,-1,-1,-1,8,8,9,9,9,9
+combustion_turbine_capacity_mw,-1,-1,-1,-1,-1,-1,9,9,10,10,10,10
+steam_capacity_mw,-1,-1,-1,-1,-1,-1,10,10,11,11,11,11
+hydro_capacity_mw,-1,-1,-1,-1,-1,-1,11,11,12,12,12,12
+wind_capacity_mw,-1,-1,-1,-1,-1,-1,12,12,13,13,13,13
+pv_capacity_mw,-1,-1,-1,-1,-1,-1,13,13,14,14,14,14
+all_storage_capacity_mw,-1,-1,-1,-1,-1,-1,14,14,15,15,15,15
+other_capacity_mw,-1,-1,-1,-1,-1,-1,15,15,16,16,16,16
+total_capacity_mw_2,-1,-1,-1,-1,-1,-1,16,16,17,17,17,17

```

Comment:
ugh, sorry you had to redo all of these! I know you've been redoing lots of them... but each time I see these it makes me think we shouldn't have even done the rough draft version before being able to test them.